tests/ui/rfc-2627-raw-dylib/dlltool-failed.stderr

```diff
@@ -0,0 +1,4 @@
+error: Dlltool could not create import library: $DLLTOOL: Syntax error in def file $DEF_FILE:1
```

Comment:
I imagine that the exact wording of the error message can change between different dlltool versions. We might need to find a more robust way of testing this, but let's give it try.

Reply:
I'm already using a regex to strip the paths from this message, so if we find that dlltool's error messages aren't stable between versions then we can extend that regex to strip out more things.